yt_dlp/extractor/xumo.py

```diff
@@ -0,0 +1,191 @@
+from .common import InfoExtractor
+from ..utils import (
+    determine_ext,
+    smuggle_url,
+    traverse_obj,
+    unsmuggle_url,
+    url_or_none
+)
+
+
+class XumoIE(InfoExtractor):
+
+    _VALID_URL = r'https?://(?:www\.)?xumo\.tv/.*/(?P<id>XM[A-Z0-9]{12})(?:/.*)?'
+    _TESTS = [{
+        # movie
+        'url': 'https://www.xumo.tv/free-movies/zombie-high/XM01OEG1MIJVJR',
+        'params': {
+            'check_formats': True
```

Comment:
why?

Reply:
I get an unauthorized error without that parameter. Not sure why exactly.